.github/workflows/build-deploy-pudl.yml

```diff
@@ -0,0 +1,109 @@
+name: build-deploy-pudl
+on:
+  push:
+    branches:
+      - gce-deploy # for testing purposes
```

Comment:
Should this still be `gce-deploy` or should it get changed to `dev` now for the merge?

Reply:
This should actually be removed. The action should only run on a tag or schedule event.